sdk/tests/common/test_signer.rs

```diff
@@ -0,0 +1,26 @@
+use c2pa::{CallbackSigner, SigningAlg};
```

Comment:
This file looks like a great example to call out for how to make your own `CallbackSigner`.